migrations/20170816230206-generatorTemplateChangeKeywordsToTags.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
```

Comment:
I'm making the same change to both Generators and GeneratorTemplates. This PR is for GeneratorTemplates, including both api and model changes. The other PR is for the Generators model change.